target/linux/ramips/image/mt7621.mk

```diff
@@ -88,6 +88,13 @@ define Device/hc5962
 endef
 TARGET_DEVICES += hc5962
 
+define Device/k2p
+  DTS := K2P
+  IMAGE_SIZE := 16056320
```

Comment:
Not sure if I understand your question. The LEDE image is written to the firmware partition. Hence, the image can not be bigger than your firmware partition. Your firmware partition has a size of 0xf60000 => 16121856 bytes => 15744KByte.